app/android/app/build.gradle

```diff
@@ -52,10 +52,10 @@ android {
 
     signingConfigs {
         debug {
-            storeFile file('key.jks')
-            storePassword 'instituteapp'
-            keyAlias 'androiddebugkey'
-            keyPassword 'instituteapp'
+            storeFile file('upload-keystore.jks')
+            storePassword 'password'
+            keyAlias 'uploadkeystore'
+            keyPassword 'password'
```

Comment:
Don't include this change in the PR as this will break authentication for other devs.